org.freedownload.Manager.yaml

```diff
@@ -0,0 +1,83 @@
+app-id: org.freedownload.Manager
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: .
+command: fdm
+tags:
+  - proprietary
+finish-args:
+  - --socket=wayland 
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-music
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-pictures
```

Comment:
it's quite many places for a downloadmanager. I think xdg-download ought to be all you need.